### PR TITLE
messages: use the outpoint in the `sig` message

### PR DESCRIPTION
**Pull request by darosior**
Quite a big overlook on this one..

I also grepped for `txid` and fixed a few inconsistencies.

Fixes https://github.com/re-vault/practical-revault/issues/62

Signed-off-by: Antoine Poinsot <darosior@protonmail.com>

### CODE REVIEW

File: transactions.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ The transaction which spends the [`vault_tx`](vault_tx) output to the EDV by the
 - count: 1
 - inputs[0]:
     - txid: `<deposit_tx txid>`
-    - vout: N/A
+    - vout: `<deposit_tx outpoint>`
```
Comment (JSwambo) [inline transactions.md:154]: I thought the [outpoint](https://btcinformation.org/en/glossary/outpoint) was the `TXID` and the `vout`? Let's either combine the `txid` and `vout` fields into an `outpoint` field, or keep them separate (to be consistent with the other transactions for which the `vout` are specified).

File: messages.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ participants.
             ...
         },
         "txid": "txid",
-        "vault_txid": "vault transaction txid"
+        "deposit_outpoint": "deposit utxo outpoint"
```
Comment (JSwambo) [inline messages.md:77]: Nice catch!

File: transactions.md (hunk 1 above)
Comment (darosior) [reply transactions.md:154]: Yep, typo here good catch :)

Comment (darosior) [reply transactions.md:154]: Went for keeping the separation

Comment (JSwambo) [inline transactions.md:154]: I think you forgot to change this one?

Comment (darosior) [reply transactions.md:154]: :sob:

**Comment by darosior** [comment]
Thanks for the sharp look, i think this one's for good!

**Review by JSwambo — APPROVED** [review]
ACK af5efd5c7c634b29f646043d354fe30704fa9504

**Pull request merged**